### PR TITLE
[16][MIG] stock_restrict_lot

### PR DESCRIPTION
**Pull request by florian-dacosta**
Standard migration
@petrus-v @sebalix @newtratip @bizzappdev

### CODE REVIEW

**Review by bizzappdev — APPROVED** [review]
Code LGTM

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_restrict_lot/tests/test_restrict_lot.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        )
+        warehouse = self.env["stock.warehouse"].search(
+            [("company_id", "=", self.env.company.id)], limit=1
+        )
+        warehouse.delivery_steps = "pick_ship"
+
+        self.product.categ_id.route_ids |= self.env["stock.route"].search(
+            [("name", "ilike", "deliver in 2")]
+        )
+        # self.env["stock.warehouse"].write(dict(delivery_steps='pick_ship',))
```
Comment (petrus-v) [inline stock_restrict_lot/tests/test_restrict_lot.py]: just realise I've let this useless line while coding this test on V14.0...

**Review by petrus-v — APPROVED** [review]

File: stock_restrict_lot/tests/test_restrict_lot.py (hunk 1 above)
Comment (florian-dacosta) [reply stock_restrict_lot/tests/test_restrict_lot.py]: Let's remove it then !

**Review by sebalix — APPROVED** [review]

**Pull request merged**